app/main.py

```diff
@@ -2,5 +2,22 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache = []
+
+    def wrapper(*args) -> None:
+        for data in cache:
+            if data["arguments"] == args:
+                print("Getting from cache")
+                return data.get("result")
```

Comment:
you can simplify the structure of the cache.

cache can be a dictionary with args as the key and the result of the function as its value.

Reply:
Yes, I understand, but how to be when the argument is a tuple? can it be a dictionary key?

Reply:
Got it, I was wrong...